scripts/crossgen/crossgen_roslyn.sh

```diff
@@ -10,15 +10,14 @@ BIN_DIR="$( cd $1 && pwd )"
 
 UNAME=`uname`
 
-if [ -z "$RID" ]; then
-    if [ "$UNAME" == "Darwin" ]; then
-        RID=osx.10.10-x64
-    elif [ "$UNAME" == "Linux" ]; then
-        RID=ubuntu.14.04-x64
-    else
-        echo "Unknown OS: $UNAME" 1>&2
-        exit 1
-    fi
+# Always recalculate the RID because the package always uses a specific RID, regardless of OS X version or Linux distro.
+if [ "$UNAME" == "Darwin" ]; then
+    RID=osx.10.10-x64
```

Comment:
No, because in this case we are calculating the package ID of the package containing crossgen and it uses `osx.10.10`. Normally, NuGet will handle the fallback, but this isn't using NuGet, it's a script so we do this a simpler way, as described in the comment.